scripts/devel/build_and_install.py

```diff
@@ -53,20 +53,26 @@ def main(argv=sys.argv[1:]):
         action='store_true',
         help='The flag if the workspace should be cleaned after the '
              'invocation')
+    parser.add_argument(
+        '--run-abichecker',
+        dest='abichecking',
```

Comment:
Nitpick: please remove the custom `dest`. The default destination `run_abichecker` is just fine.

Reply:
ff5cffd